### DISCUSSION TITLE
What's the best way to handle i18n in Next?

### DISCUSSION THREAD

**Discussion by justintemps** [vercel/next.js#17918] [Help] [answered]
I've already tried [next-i18next](https://github.com/isaachinman/next-i18next) but I'm having a lot of trouble with it and was hoping there might be some other options out there.

All I really need is something to manage the routing and build out the different language versions for SSR as I'm going to get all of the translations from Prismic.

**Comment by ijjk** [comment]
Hi, have you taken a look at our i18n routing RFC here https://github.com/vercel/next.js/discussions/17078?

**Reply by justintemps** [reply]
I have! Super exciting, when is that expected to land?

**Comment by ljosberinn** [comment, marked as answer]
Can recommend [next-translate](https://github.com/vinissimus/next-translate) and built a similar solution until the RFC lands.

**Reply by justintemps** [reply]
Just was I was looking for. Thank you!
